### ISSUE TITLE
exceptions option for vue/no-multi-spaces

### ISSUE THREAD

**Issue by rjgotten** [vuejs/eslint-plugin-vue#535]
**Tell us about your environment**

* **ESLint Version:** 5.0.1
* **eslint-plugin-vue Version:** 4.5.0
* **Node Version:** 8.11.3 (LTS)

**Please show your full configuration:**

```
module.exports = {
  root: true,
  extends: "plugin:vue/recommended",
  parser: "vue-eslint-parser",
  parserOptions: {
    parser: "babel-eslint",
    sourceType: "module",
    ecmaVersion: 8
  },
  env: {
    browser: true,
    es6: true
  },
  plugins: ["vue"]
};
```

**What did you do? Please include the actual source code causing the issue.**

```vue
<i
  v-if="useIconArrow"
  :class="{
    'fa-angle-up'   : isExpanded,
    'fa-angle-down' : !isExpanded,
  }"
  class="fal"
></i>
```

**What did you expect to happen?**

No validation error from `vue/no-multi-spaces`, which is meant to handle unnecessary spaces on the template side and should not bother checking inside JS expressions. (That should be the domain of 'normal' ESLint rules.)

**What actually happened? Please include the actual, raw output from ESLint.**

`vue/no-multi-spaces` gives an error on an object literal inside a JS expression, which aligns property assignments on the colon:

```
error  Multiple spaces found before ':'  vue/no-multi-spaces
```

**Comment by mysticatea** [comment]
Thank you for this issue.

This is intentional behavior rather than false positive.
I think that we can consider `exceptions` option like core [no-multi-spaces](https://eslint.org/docs/rules/no-multi-spaces) rule.
